web-client/src/ducks/requests/actions.ts

```diff
@@ -3,37 +3,49 @@ import { IRequest, Request } from 'src/models/requests';
 /* TODO: why rename? */
 import {
   createUserRequest,
-  getAcceptedPost as getAcceptedPostFunc,
-  getOpenPost as getOpenPostFunc,
+  getCavRequestPosts as getCavPostFunc,
+  getFindPosts as getFindPostsFunc,
+  getPinReqestPosts as getPinRequestPostsFunc,
   setUserRequest,
 } from './functions';
 import {
   CHANGE_MODAL,
-  GET_OFFER_POST,
-  GET_REQUEST_POST,
+  GET_CAV_REQUEST_POSTS,
+  GET_FIND_REQUEST_POSTS as GET_FIND_POSTS,
+  GET_PIN_REQUEST_POSTS,
   IgetRequestPosts,
-  RESET_OFFER_POST,
-  RESET_REQUEST_POST,
+  RESET_CAV_REQUEST_POSTS,
+  RESET_FIND_REQUEST_POSTS,
+  RESET_PIN_REQUEST_POSTS,
   RESET_SET,
   SET,
   SET_TEMP_REQUEST,
 } from './types';
 
-export const getRequestPosts = (payload: IgetRequestPosts) => (
+export const getFindPosts = (payload: IgetRequestPosts) => (
```

Comment:
what is a find post? How is this different from CAVRequestPosts and PINRequestPosts

Reply:
getFindPosts is used for getting posts for the map.

Reply:
This function was doing double duty - it was called both by find and by my requests.  It is no longer needed by my requests.  I can delete it in the next pull request.